### ISSUE TITLE
Add CLI option for `addNewValuesAsEmpty`

### ISSUE THREAD

**Issue by zjx20** [Flinesoft/BartyCrouch#2]
`addNewValuesAsEmpty` is set to true by default. It would be good if there was an option to turn it off.

**Comment by Dschee** [comment]
Hi @zjx20, thanks for asking this question. As you might have guessed from code I initially planned to add this option via the CLI but finally opted not to do so because I wasn't sure about how useful it would be. But now that you're asking for it, sure – I just released a new version (1.2.0) which adds this feature.

I hope this helps!

**Comment by Dschee** [comment]
Ah, it doesn't seem to work as of yet. I'll have a look tomorrow again. I'll reopen until then.

**Comment by zjx20** [comment]
Thanks for the quick response!
By the way, is it possible to keep comments in sync?

For example, there is one generated comment:
```
/* Class = "UITableViewSection"; headerTitle = "Foo"; ObjectID = "HQs-Z9-d7P"; */
"HQs-Z9-d7P.headerTitle" = "Foo";
```

If the "Foo" value was changed to "Bar" in the base Main.storyboard, can `.strings` files be also updated to reflect the change? To become:
```
/* Class = "UITableViewSection"; headerTitle = "Bar"; ObjectID = "HQs-Z9-d7P"; */
"HQs-Z9-d7P.headerTitle" = "Bar";  /* gets the update too if it wasn't a translated value */
```

**Comment by Dschee** [comment]
I'm now investigating this issue.

Thanks also for the idea with updating the comment – could you please open a separate issue on that so we can discuss it independently? Thanks!

**Comment by zjx20** [comment]
Sure! I have opened a new issue #4 for that.

**Comment by Dschee** [comment]
:+1:

**Comment by Dschee** [comment]
Just found the issue why v1.2.0 didn't work, fixed it and released v1.2.1. This should work now!
